src/implementations/vanilla/src/components/modal/modal.scss

```diff
@@ -32,27 +33,27 @@ $shadow-layer: -1;
   flex-direction: column;
   top: 128px;
   max-width: calc(100vw - 32px);
-  width: 500px;
+  width: 600px;
   left: 50%;
   transform: translateX(-50%);
   max-height: calc(100vh - 256px);
   overflow: hidden;
 
-  border: 1px solid color(hig-gray-50);
+  border: 3px solid color(hig-blue-20);
   background: color(hig-white);
   border-radius: 4px;
   outline: none;
-  box-shadow: 0 2px 8px rgba(0, 0, 0, 0.5);
+  box-shadow: 0 0 8px 0 #BEC8D2;
```

Comment:
The color specified for the box-shadow needs some transparency to read as a shadow. I found this value in the Sketch file:

```
box-shadow: 0 0 8px 0 rgba(10, 19, 28, 60);
````

Reply:
This actually computes the same value as what's there.  I do like how it's more explicit so I've updated to use it in 8f31f7c